spread.yaml

```diff
@@ -60,6 +60,11 @@ path: /home/gopath/src/github.com/snapcore/snapd
 exclude:
     - .git
 
+debug: |
+    # something went wrong in the test, get some logs
+    echo "Snapd debug log:"
```

Comment:
Also, no need for the echo here, as this script is also executed in trace mode. We'll see the shell lines being executed before their output as usual for tasks.